helm/values/csi-driver/v1.1.0-beta/values_3parprimera.yaml

```diff
@@ -0,0 +1,58 @@
+# Default values for hpe-csi-storage.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+images:
+  # csi-driver
+  csiDriverImage: hpestorage/csi-driver:v1.1.0-beta
+  # csp-hpe
+  cspImage: dockerciuser/hpe_3par_primera_csp:test
```

Comment:
@bhagyashree-sarawate NOTE: This would be changed once we put the csp image under hpestorage repo.

Reply:
done